hudi-cli/src/main/java/org/apache/hudi/cli/commands/SparkMain.java

```diff
@@ -353,14 +353,20 @@ private static int deduplicatePartitionPath(JavaSparkContext jsc, String duplica
 
   private static int doBootstrap(JavaSparkContext jsc, String tableName, String tableType, String basePath,
```

Comment:
The original parameter list is not very graceful. I suggest we can only receive `args`, then use a builder to build(extract) the metadata of the bootstrap action. wdyt?

Reply:
yes, filed another ticket to rack this : https://issues.apache.org/jira/browse/HUDI-2104